libraries/loader/README.md

```diff
@@ -150,18 +152,11 @@ particularly useful for getting a fresh component between tests.
 
 We generally recommend one component loader per project, and that you expose
 it in an executable such that you do `node server.js <component>` to start a
-process running the specified component.  A handy way to run the loader on startup,
-given both the profile ($NODE_ENV) and process (target component):
+process running the specified component.
 
 ```js
 // If this file is executed launch component from first argument
 if (!module.parent) {
-  load(process.argv[2], {
-    process: process.argv[2],
-    profile: process.env.NODE_ENV,
-  }).catch(err => {
-    console.log(err.stack);
-    process.exit(1);
-  });
+  load(process.argv[2]);
```

Comment:
This needs a catch, too.

I kind of like the idea of some syntax to say "this is a top-level invocation of the loader" that will automatically handle errors, as it will remove another 2-3 lines of repetitive code from the services.  But this works too!

Reply:
Yeah, I would like that eventually too but this was easier to change for now. Created [bug 1527969](https://bugzilla.mozilla.org/show_bug.cgi?id=1527969) for this.